### ISSUE TITLE
代码app.py第413行parsing_out = parsing_net(img_path=image_raw, bbox=None)运行失败

### ISSUE THREAD

**Issue by LSliu666** [aigc3d/LHM#114]
cuda11.8运行app.py在413行parsing_out = parsing_net(img_path=image_raw, bbox=None)报错
报错内容为
BiRefNet is ready to use.
Segmentation fault (core dumped)

不知道什么原因，希望能够指点一下
环境
Package                           Version
--------------------------------- ------------------
absl-py                           2.2.2
accelerate                        1.6.0
addict                            2.4.0
aiofiles                          23.2.1
aiohappyeyeballs                  2.6.1
aiohttp                           3.11.18
aiosignal                         1.3.2
airportsdata                      20250224
annotated-types                   0.7.0
antlr4-python3-runtime            4.9.3
anyio                             4.9.0
astor                             0.8.1
asttokens                         3.0.0
async-timeout                     4.0.3
attrs                             25.3.0
basicsr                           1.4.2
bcrypt                            4.3.0
blake3                            1.0.4
blinker                           1.9.0
boto3                             1.38.2
botocore                          1.38.2
certifi                           2025.1.31
cffi                              1.17.1
charset-normalizer                3.4.1
chumpy                            0.70
click                             8.1.7
coloredlogs                       15.0.1
comm                              0.2.2
ConfigArgParse                    1.7
cryptography                      44.0.2
cupy-cuda12x                      13.4.0
cycler                            0.12.1
dash                              2.17.1
dash-core-components              2.0.0
dash-html-components              2.0.0
dash-table                        5.0.0
dataclasses-json                  0.6.7
decorator                         5.2.1
decord                            0.6.0
Deprecated                        1.2.18
depyf                             0.18.0
diff_gaussian_rasterization       0.0.0
diffusers                         0.32.0
dill                              0.3.9
diskcache                         5.6.3
dna                               0.0.1
dnspython                         2.7.0
einops                            0.8.1
email_validator                   2.2.0
exceptiongroup                    1.2.2
executing                         2.2.0
facexlib                          0.3.0
fastapi                           0.112.4
fastapi-cli                       0.0.7
fastjsonschema                    2.21.1
fastrlock                         0.8.3
ffmpy                             0.5.0
filelock                          3.18.0
filterpy                          1.4.5
Flask                             3.0.3
flatbuffers                       25.2.10
fonttools                         4.57.0
freetype-py                       2.5.1
frozenlist                        1.6.0
fsspec                            2025.3.2
future                            1.0.0
gfpgan                            1.3.8
gguf                              0.10.0
GPUtil                            1.4.0
gradio                            4.43.0
gradio_client                     1.3.0
grpcio                            1.71.0
gsplat                            1.4.0
h11                               0.16.0
httpcore                          1.0.9
httptools                         0.6.4
httpx                             0.28.1
huggingface-hub                   0.23.2
humanfriendly                     10.0
hydra-core                        1.3.2
idna                              3.10
imageio                           2.34.1
imageio-ffmpeg                    0.6.0
importlib_metadata                8.6.1
importlib_resources               6.5.2
iniconfig                         2.0.0
interegular                       0.3.3
iopath                            0.1.10
ipython                           8.35.0
ipywidgets                        8.1.6
itsdangerous                      2.2.0
jaxtyping                         0.2.38
jedi                              0.19.2
Jinja2                            3.1.6
jiter                             0.9.0
jmespath                          1.0.1
joblib                            1.4.2
jsonschema                        4.23.0
jsonschema-specifications         2024.10.1
jupyter_core                      5.7.2
jupyterlab_widgets                3.0.14
kiui                              0.2.14
kiwisolver                        1.4.8
kornia                            0.7.2
kornia_rs                         0.1.8
lark                              1.2.2
lazy_loader                       0.4
lightning-utilities               0.14.3
llvmlite                          0.43.0
lm-format-enforcer                0.10.11
lmdb                              1.6.2
loguru                            0.7.3
lpips                             0.1.4
Markdown                          3.8
markdown-it-py                    3.0.0
MarkupSafe                        2.1.5
marshmallow                       3.26.1
matplotlib                        3.5.3
matplotlib-inline                 0.1.7
mdurl                             0.1.2
megfile                           4.1.0.post2
mistral_common                    1.5.4
modelscope                        1.25.0
mpmath                            1.3.0
msgpack                           1.1.0
msgspec                           0.19.0
multidict                         6.4.3
mypy_extensions                   1.1.0
nbformat                          5.10.4
nest-asyncio                      1.6.0
networkx                          3.4.2
ninja                             1.11.1.4
numba                             0.60.0
numpy                             1.23.0
nvidia-cublas-cu11                11.11.3.6
nvidia-cuda-cupti-cu11            11.8.87
nvidia-cuda-nvrtc-cu11            11.8.89
nvidia-cuda-runtime-cu11          11.8.89
nvidia-cudnn-cu11                 8.7.0.84
nvidia-cufft-cu11                 10.9.0.58
nvidia-curand-cu11                10.3.0.86
nvidia-cusolver-cu11              11.4.1.48
nvidia-cusparse-cu11              11.7.5.86
nvidia-nccl-cu11                  2.20.5
nvidia-nvtx-cu11                  11.8.86
objprint                          0.3.0
omegaconf                         2.3.0
onnxruntime-gpu                   1.18.1
open3d                            0.19.0
openai                            1.66.3
opencv-python                     4.11.0.86
opencv-python-headless            4.11.0.86
orjson                            3.10.16
outlines                          0.1.11
outlines_core                     0.1.26
packaging                         25.0
pandas                            2.2.3
paramiko                          3.5.1
parso                             0.8.4
partial-json-parser               0.2.1.1.post5
pexpect                           4.9.0
pillow                            10.4.0
pip                               25.0.1
platformdirs                      4.3.7
plotly                            5.22.0
pluggy                            1.5.0
plyfile                           1.1
pooch                             1.8.2
portalocker                       3.1.1
prometheus_client                 0.21.1
prometheus-fastapi-instrumentator 7.0.2
prompt_toolkit                    3.0.51
propcache                         0.3.1
protobuf                          6.30.2
psutil                            5.9.8
ptyprocess                        0.7.0
pure_eval                         0.2.3
py-cpuinfo                        9.0.0
pybind11                          2.13.6
pycountry                         24.6.1
pycparser                         2.22
pydantic                          2.10.6
pydantic_core                     2.27.2
pydub                             0.25.1
pyglet                            2.1.5
pygltflib                         1.16.2
Pygments                          2.19.1
PyMatting                         1.1.13
PyNaCl                            1.5.0
PyOpenGL                          3.1.0
pyparsing                         3.2.3
pyquaternion                      0.9.9
pyrender                          0.1.45
pytest                            8.3.5
python-dateutil                   2.9.0.post0
python-multipart                  0.0.20
pytorch-lightning                 2.5.1
pytorch3d                         0.7.8
pytz                              2025.2
PyYAML                            6.0.1
referencing                       0.36.2
regex                             2024.11.6
rembg                             2.0.63
requests                          2.32.3
retrying                          1.3.4
rich                              14.0.0
rich-toolkit                      0.13.2
roma                              1.5.2.1
rpds-py                           0.23.1
ruff                              0.11.7
s3transfer                        0.12.0
safetensors                       0.5.3
SAM-2                             1.0
scikit-image                      0.24.0
scikit-learn                      1.6.1
scipy                             1.13.1
semantic-version                  2.10.0
sentencepiece                     0.2.0
setuptools                        74.0.0
shellingham                       1.5.4
simple_knn                        1.0.0
six                               1.17.0
smplx                             0.1.28
sniffio                           1.3.1
spaces                            0.35.0
stack-data                        0.6.3
starlette                         0.38.6
sympy                             1.13.1
taming-transformers-rom1504       0.0.6
tb-nightly                        2.20.0a20250424
tenacity                          8.4.2
tensorboard-data-server           0.7.2
threadpoolctl                     3.5.0
tifffile                          2025.3.30
tiktoken                          0.9.0
timm                              1.0.15
tokenizers                        0.19.1
tomli                             2.2.1
tomlkit                           0.12.0
torch                             2.3.0+cu118
torchaudio                        2.3.0+cu118
torchmetrics                      1.7.1
torchvision                       0.18.0+cu118
tqdm                              4.66.4
traitlets                         5.14.3
transformers                      4.41.2
trimesh                           4.4.9
triton                            2.3.0
typeguard                         2.13.3
typer                             0.15.2
typing_extensions                 4.13.2
typing-inspect                    0.9.0
tzdata                            2025.2
urllib3                           2.4.0
uvicorn                           0.34.0
uvloop                            0.21.0
varname                           0.14.0
wadler_lindig                     0.1.5
watchfiles                        1.0.4
wcwidth                           0.2.13
websockets                        12.0
Werkzeug                          3.0.6
wheel                             0.45.1
widgetsnbextension                4.0.14
wrapt                             1.17.2
xatlas                            0.0.9
xformers                          0.0.26.post1+cu118
yapf                              0.43.0
yarl                              1.20.0
zipp                              3.21.0

**Comment by lingtengqiu** [comment]
感觉是版本问题？ 你有cuda 吗？

**Comment by LSliu666** [comment]
是SAM2Seg类的问题吗？我下载模型使用的是这条命令
wget https://virutalbuy-public.oss-cn-hangzhou.aliyuncs.com/share/aigc3d/data/LHM/LHM_prior_model.tar 
tar -xvf LHM_prior_model.tar

**Comment by LSliu666** [comment]
将cuda版本改为12.1可成功运行，感谢
